src/common/runner/runner.go

```diff
@@ -47,6 +49,9 @@ func (r *Runner) Init(commands *clioptions.TagOptions) error {
 		tagGroup.InitTagGroup(dir, commands.SkipTags)
 		if simpleTagGroup, ok := tagGroup.(*simple.TagGroup); ok {
 			simpleTagGroup.SetTags(extraTags)
+		} else if externalTagGroup, ok := tagGroup.(*external.TagGroup); ok && commands.ConfigFile != "" {
+			externalTagGroup.InitConfigFile(commands.ConfigFile)
+			externalTagGroup.InitExternalTagGroup()
```

Comment:
@tronxd Why split these methods? Can the runner only know 1 init method, and let the externalTagGroup manage this?

Reply:
@nimrodkor tagGroup.InitTagGroup only accepts the root dir and the skipped tags. do we want to add a variadic "data" argument to include (in our case) the config path?